src/c4/test/tstProcessor_Group.cc

```diff
@@ -52,7 +53,7 @@ void tstProcessor_Group(rtt_dsxx::UnitTest &ut) {
     vector<double> myvec;
     size_t const vlen(5);
     for (size_t i = 0; i < vlen; ++i)
-      myvec.push_back(pid * 1000.0 + i);
+      myvec.push_back(static_cast<double>(pid * 1000 + i));
```

Comment:
This is ugly.  Learn how to enable `-Wconversion` w/o warning on `int --> double` promotion when multiplying (if this is even possible).

Reply:
The `uint64_t` type is exact, but when converted to a double it might loose its _exactness_.